### ISSUE TITLE
Error 8007007e while installing npcap

### ISSUE THREAD

**Issue by teamango** [nmap/nmap#1982]
I tried two versions: _0.9986_ and _0.9989_, but nothing worked. I also done all my pending updates on Windows 10. 
I need **Npcap** for **Wireshark**. I searched all over internet and I see that Winpcap could work too, but it didn't work either. 

I really don't know what to do anymore. 
![fdfdef](https://user-images.githubusercontent.com/33987689/77848833-3dab6380-71d0-11ea-9e57-203e1900ced4.PNG)

![gg](https://user-images.githubusercontent.com/33987689/77848837-4308ae00-71d0-11ea-82b9-99d4ca44b2e3.PNG)

**Comment by mmaraa** [comment]
Same problem here. I need npcap for Azure ATP Sensor for physical machine that has a NIC Teaming configured. Failed to create npcap service :(

**Comment by guyharris** [comment]
See also #1401.

**Comment by mmaraa** [comment]
We solved our problem. There wasn't updated root certificates on certificate store, so driver signing certificate wasn't trusted. You can see it by unzipping .exe e.g. with 7-zip and looking the certification path under certificate Insecure-EV in $PLUGINSDIR.

**Comment by guyharris** [comment]
So does Microsoft anywhere indicate that an error code of 8007007e (is that an NT status?  It has the look of a VMS-style error, so it might be) indicates something as specific as a lack of trust in a certificate?  And, if so, what code needs to be updated to suggest that rather than a generic "that failed" error?

**Comment by mmaraa** [comment]
Nope. I figured out it myself. My computers where I was installing npcap to are offline disconnected servers from the internet. There wasn't either updated Trusted root certification list on those computers however there are other updates in place.

**Comment by hummelm10** [comment]
I am having the same issue on Windows 10 Enterprise. When I try and install the drivers manually from the C:\Program Files\Npcap folder I get the following (I also have similar results to issue #1746):

```
C:\Program Files\Npcap>C:\Windows\System32\pnputil.exe -i -a *.inf
Microsoft PnP Utility

Processing inf :            npcap.inf
Successfully installed the driver.
Driver package added successfully.
Published name :            oem11.inf

Processing inf :            npcap_wfp.inf
Adding the driver package failed : The hash for the file is not present in the specified catalog file. The file is likely corrupt or the victim of tampering.


Total attempted:              2
Number successfully imported: 1

```
